packages/components/src/components/Search/Search.tsx

```diff
@@ -1,71 +1,81 @@
 import { MaterialIcons } from "@expo/vector-icons";
 import React, { useEffect, useRef, useState } from "react";
-import { Animated, View, TextInput } from "react-native";
+import { Platform, ScrollView, TextInput, View } from "react-native";
+import Animated, {
```

Comment:
Not sure why this is happening. By reading the react-native-reanimated documentation, this seems to be the correct way to import Animated. I tried updating the react-native-reanimated package to the latest version as well (3.6.2), and added web support by installing @babel/plugin-proposal-export-namespace-from, but i'm still getting the same error. Any idea on how to prevent this error?